lib/uksched/include/uk/sched.h

```diff
@@ -233,7 +233,7 @@ void uk_sched_thread_exit(void) __noreturn;
 void uk_sched_thread_exit2(uk_thread_gc_t gc_fn, void *gc_argp) __noreturn;
 
 /* Terminates another thread */
-void uk_sched_thread_terminate(struct uk_thread *thread);
+void uk_sched_thread_terminate(struct uk_thread *thread) __nonnull;
```

Comment:
The function already uses `UK_ASSERT`, using `__nonnull` is redundant here and would generate avoidable warnings